### PR TITLE
Fixes #7469: /etc/init.d/rudder-jetty restart does not restart jetty

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7469

### CODE REVIEW

File: rudder-jetty/SOURCES/Makefile [hunk 1]
```diff
@@ -50,8 +50,9 @@ localdepends: ./jetty7
 	patch -p0 -s < jetty-init-stop-fix.patch
 	patch -p0 -s < jetty-init-sizecheck.patch
 	patch -p0 -s < jetty-init-use-rudder-jetty-defaults.patch
-	patch -p0 -s < jetty-init-prevent-false-failed-starts.patch
+	patch -p0 -s < patches/jetty/jetty-init-prevent-false-failed-starts.patch
```
Comment (jooooooon) [inline rudder-jetty/SOURCES/Makefile]: I moved this file to ease merges into the upper branches.

File: rudder-jetty/SOURCES/patches/jetty/jetty-init-stop-forcestop.patch [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
++    PS_COMMAND="/bin/vzps -E 0"
++  else # use rudder provided vzps
++    PS_COMMAND="/opt/rudder/bin/vzps.py -E 0"
++  fi  
++elif [ -n "${ns}" ]; then # we have namespaces
++  # the sed is here to prepend a fake user field that is removed by the -o option (it is never used)
++  PS_COMMAND="eval ps --no-header -e -O utsns | grep -E '^[[:space:]]*[[:digit:]]*[[:space:]]+${ns}' | sed 's/^/user /'"
++else # standard unix
++  PS_COMMAND="ps -ef"
++fi
```
Comment (jooooooon) [inline rudder-jetty/SOURCES/patches/jetty/jetty-init-stop-forcestop.patch]: This is copied from rudder-agent.init

File: rudder-jetty/SOURCES/Makefile (hunk 1 above)
Comment (peckpeck) [inline rudder-jetty/SOURCES/Makefile]: :+1:

**Comment by peckpeck** [comment]
OK, merging

**Pull request merged**